### PR TITLE
Adding author, acknowledgments, citation

### PR DESCRIPTION
**Pull request by hkaiser**

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -72,10 +76,10 @@ HPX is currently funded by
   any information, apparatus, product, or process disclosed, or represents that
   its use would not infringe privately owned rights.
 
-* The Bavarian Research Foundation (Bayerische Forschungsstiftung) through the
+- The Bavarian Research Foundation (Bayerische Forschungsstiftung) through the
   grant AZ-987-11.
 
-* The European Commission's Horizon 2020 programme through the grant
+- The European Commission's Horizon 2020 programme through the grant
```
Comment (sithhell) [inline paper.md:82]: I guess the two above are not active anymore. So hpx is technically not currently funded by those.

Comment (hkaiser) [reply paper.md:82]: We might want to rephrase this by saying 'has been funded'.

Comment (diehlpk) [reply paper.md:82]: Like that

File: paper.md [hunk 2]
```diff
@@ -31,24 +34,25 @@ bibliography: paper.bib
...
+National Supercomputing Centre (CSCS), and the Department of Computer Science 3 
+- Computer Architecture at the University of Erlangen Nuremberg who fund and 
+support our work.
 
 We would also like to thank the following organizations for granting us
 allocations of their compute resources: LSU HPC, LONI, XSEDE, NERSC, CSCS/ETHZ,
 and the Gauss Center for Supercomputing.
 
-HPX is currently funded by
+HPX is currently funded by:
```
Comment (hkaiser) [inline paper.md]:
```suggestion
HPX has been funded by:
```

**Pull request merged**